src/main/io/serial_cli.c

```diff
@@ -1214,24 +1210,27 @@ static void cliRxFail(char *cmdline)
     }
 }
 
-static void printAux(uint8_t dumpMask, const modeActivationProfile_t *defaultModeActivationProfile)
+static void printAux(uint8_t dumpMask, const modeActivationProfile_t *modeActivationProfile, const modeActivationProfile_t *defaultModeActivationProfile)
 {
+    const char *format = "aux %u %u %u %u %u\r\n";
     // print out aux channel settings
     for (uint32_t i = 0; i < MAX_MODE_ACTIVATION_CONDITION_COUNT; i++) {
-        const modeActivationCondition_t *mac = &modeActivationProfile()->modeActivationConditions[i];
-        const modeActivationCondition_t *macDefault = &defaultModeActivationProfile->modeActivationConditions[i];
-        const bool equalsDefault = mac->modeId == macDefault->modeId
-            && mac->auxChannelIndex == macDefault->auxChannelIndex
-            && mac->range.startStep == macDefault->range.startStep
-            && mac->range.endStep == macDefault->range.endStep;
-        const char *format = "aux %u %u %u %u %u\r\n";
-        cliDefaultPrintf(dumpMask, equalsDefault, format,
-            i,
-            macDefault->modeId,
-            macDefault->auxChannelIndex,
-            MODE_STEP_TO_CHANNEL_VALUE(macDefault->range.startStep),
-            MODE_STEP_TO_CHANNEL_VALUE(macDefault->range.endStep)
-        );
+        const modeActivationCondition_t *mac = &modeActivationProfile->modeActivationConditions[i];
+        bool equalsDefault = true;
```

Comment:
Well, I tested it the other way. And OK, you are right, it does not matter. `equalsdefaults` will be set true/false by evaluation later on anyhow, when pointer is not NULL. To me a default value of `false` only felt more natural. But OK, never mind me...